cmd/list/firmware.go

```diff
@@ -27,7 +40,18 @@ var listFirmware = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		firmware, _, err := client.ListServerComponentFirmware(ctx, nil)
+		modelSlice := make([]string, 0)
+		if flagsDefinedListFirmware.model != "" {
+			modelSlice = append(modelSlice, strings.Split(flagsDefinedListFirmware.model, ",")...)
```

Comment:
The `model` field here is a string, not a slice. I think if you want to support a comma-delimited options, instead of rolling your own try: [cobra/viper's built in support](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.StringSlice).

Reply:
oh perfect. Will do.